sebs/knative/triggers.py

```diff
@@ -0,0 +1,107 @@
+import concurrent.futures
+import datetime
+import json
+import subprocess
+from typing import Dict, List, Optional
+
+from sebs.faas.function import ExecutionResult, Trigger
+
+
+class KnativeLibraryTrigger(Trigger):
+    def __init__(self, fname: str, func_cmd: Optional[List[str]] = None):
+        super().__init__()
+        self.fname = fname
+        if func_cmd:
+            self._func_cmd = [*func_cmd, "invoke", "--target", "remote"]
```

Comment:
We are repeating this on line 28 - can we only store `*func_cmd` inside `self._func_cmd` and build/append everything else at the call site (line 36)?